### PR TITLE
add-cancel-button-and-simple-store

### PR DESCRIPTION
**Pull request by OlexxandrS**

### CODE REVIEW

**Comment by OlexxandrS** [comment]
[Implement the "Cancel" button](https://trello.com/c/pFcViljE/9-implement-the-cancel-button)

File: src/data/payments.js [hunk 1]
```diff
@@ -1,44 +0,0 @@
-const payments = [
```
Comment (filkovsp) [inline src/data/payments.js:1]: hi Alex 
why do we have an empty file?

File: src/components/Main.js [hunk 2]
```diff
@@ -1,109 +1,141 @@
...
+  const [foreingCurrency, setForeingCurrency] = useState("USD");
+  const [foreingAmount, setForeingAmount] = useState(0);
+  const [homeAmount, setHomeAmount] = useState("???");
 
-    function convert(){
-        fetch(`https://api.exchangerate.host/convert?from=${foreingCurrency}&to=GBP&amount=${foreingAmount}`)
-        .then(response => response.json())
-        .then(data => setHomeAmount(data.result));
-    }
+  const [, updateState] = React.useState();
```
Comment (filkovsp) [inline src/components/Main.js:24]: Hi 
a bunch of lines above have this style of state initialization:
```js
const [....] = useState(...);
```
but other 2 lines below are different:
```js
const [....] = React.useState(...);
```
let's be more consistent here please, shall we? :)

File: src/data/payments.js (hunk 1 above)
Comment (OlexxandrS) [reply src/data/payments.js:1]: this file is not needed at the moment, but it will be needed later)

File: src/components/Main.js (hunk 2 above)
Comment (OlexxandrS) [reply src/components/Main.js:24]: my bad

File: src/components/Main.js [hunk 3]
```diff
@@ -1,109 +1,141 @@
...
-import './Main.css';
+import store, { cancelPayment } from "../data/store";
+import "./Main.css";
 
+function Main() {
+  const [foreingCurrency, setForeingCurrency] = useState("USD");
+  const [foreingAmount, setForeingAmount] = useState(0);
+  const [homeAmount, setHomeAmount] = useState("???");
+  const homeCurrency = "GBP";
+  const [, updateState] = React.useState();
```
Comment (filkovsp) [inline src/components/Main.js:24]: let's make also this line consistent with the common style in our project: 
```js
const [, updateState] = useState();
```
in latest react versions we don't even need to import the React component :)

**Review by filkovsp — DISMISSED** [review]
good job

**Comment by OlexxandrS** [comment]
need review

**Review by filkovsp — APPROVED** [review]
cool! great job!

**Pull request merged**